sdk/communication/communication-administration/src/common/tracing.ts

```diff
@@ -1,57 +1,13 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { OperationOptions } from "@azure/core-http";
-import { getTracer } from "@azure/core-tracing";
-import { Span, SpanOptions, SpanKind } from "@opentelemetry/api";
-
-type OperationTracingOptions = OperationOptions["tracingOptions"];
+import { createSpanFunction } from "@azure/core-tracing";
 
 /**
  * Creates a span using the global tracer.
  * @internal
- * @param name - The name of the operation being performed.
- * @param tracingOptions - The options for the underlying http request.
  */
-export function createSpan<T extends OperationOptions>(
-  operationName: string,
-  operationOptions: T
-): { span: Span; updatedOptions: T } {
-  const tracer = getTracer();
-  const tracingOptions = operationOptions.tracingOptions || {};
-  const spanOptions: SpanOptions = {
-    ...tracingOptions.spanOptions,
-    kind: SpanKind.INTERNAL
-  };
-
-  const span = tracer.startSpan(`Azure.Communication.${operationName}`, spanOptions);
-
-  span.setAttribute("az.namespace", "Microsoft.Communication");
-
-  let newSpanOptions = tracingOptions.spanOptions || {};
-  if (span.isRecording()) {
-    newSpanOptions = {
-      ...tracingOptions.spanOptions,
-      parent: span.context(),
-      attributes: {
-        ...spanOptions.attributes,
-        "az.namespace": "Microsoft.Communication"
-      }
-    };
-  }
-
-  const newTracingOptions: OperationTracingOptions = {
-    ...tracingOptions,
-    spanOptions: newSpanOptions
-  };
-
-  const newOperationOptions: T = {
-    ...operationOptions,
-    tracingOptions: newTracingOptions
-  };
-
-  return {
-    span,
-    updatedOptions: newOperationOptions
-  };
-}
+export const createSpan = createSpanFunction({
```

Comment:
@0rland0Wats0n , @DominikMe  - can I get your sign-off on these telemetry changes I've made for the communications packages? We're centralizing the code to create spans into core-tracing. I think you probably just used the "standard" snippet so this should just be a free code reduction :)